examples/step-2/step-2.cc

```diff
@@ -85,13 +85,13 @@ void make_grid (Triangulation<2> &triangulation)
                outer_radius = 1.0;
   GridGenerator::hyper_shell (triangulation,
                               center, inner_radius, outer_radius,
-                              10);
+                              5 );
```

Comment:
A coarser mesh makes the matrix smaller, which makes the pixels in the pattern much easier to see.